### DISCUSSION TITLE
ssh tunnel doens't working after logstash docker update

### DISCUSSION THREAD

**Discussion by enzodevs2000** [telekom-security/tpotce#1319] [Q&A] [answered]
Hi, i have a cron on my HIVE to auto update the docker images. Yesterday, a new version of logstash image was released. Today, receive of logs on HIve and Kibana has stopped.

OS: Debian 11
T-pot 22.04 Post Install

Using `ps -aux | grep sshd` doesn't show the connections with the sensors

I would like to know if anyone else is experiencing the same problem.

**Comment by rocbr** [comment]
I couldn't identify exactly where the problem is occurring, but apparently it's an issue with the logstash docker image, as it is not being able to perform the SSH tunnel with the HIVE server. Maybe something is breaking even before the ssh tunnel is performed. In fact, there was a logstash docker image update yesterday https://hub.docker.com/r/dtagdevsec/logstash/tags.

Glances command shows logstash restarting all the time.

**Reply by t3chn0m4g3** [reply]
Thank you, I will look into it.

**Reply by t3chn0m4g3** [reply]
I cannot reproduce it, can you send me the logstash startup logs please `docker logs logstash`. Thanks. @enzodevs2000 @rocbr Based on your feedback there seem to be different reasons.
I reviewed the changes again and it was only related to ENVs that will not affect ssh connections.
But I need the logs to see what's happening on your ends.

**Reply by rocbr** [reply]
05/03 20:17:43 [NOTICE] Download complete: /etc/listbot/cve.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
0b3936|OK  |    90KiB/s|/etc/listbot/cve.yaml.bz2

Status Legend:
(OK):download completed.

05/03 20:17:43 [NOTICE] Downloading 1 item(s)
[#927849 0B/1.6MiB(0%) CN:1 DL:30KiB ETA:56s]
[#927849 400KiB/1.6MiB(23%) CN:1 DL:313KiB ETA:4s]

05/03 20:17:45 [NOTICE] Download complete: /etc/listbot/iprep.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
927849|OK  |   0.9MiB/s|/etc/listbot/iprep.yaml.bz2

Status Legend:
(OK):download completed.

Distributed T-Pot setup, sending T-Pot logs to yyyyyy.XXXXXXXX.net.

T-Pot type: SENSOR
Keyfile used: /data/elk/logstash/Sidney-Australia
Hive username: Sidney-Australia
Hive IP: yyyyyy.XXXXXXXX.net

Connection to Listbot looks good, now downloading latest translation maps.

05/03 20:18:47 [NOTICE] Downloading 1 item(s)
[#68ca65 0B/23KiB(0%) CN:1 DL:29KiB]

05/03 20:18:48 [NOTICE] Download complete: /etc/listbot/cve.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
68ca65|OK  |    88KiB/s|/etc/listbot/cve.yaml.bz2

Status Legend:
(OK):download completed.

05/03 20:18:48 [NOTICE] Downloading 1 item(s)
[#afebb4 0B/1.6MiB(0%) CN:1 DL:29KiB ETA:58s]
[#afebb4 400KiB/1.6MiB(23%) CN:1 DL:303KiB ETA:4s]

05/03 20:18:51 [NOTICE] Download complete: /etc/listbot/iprep.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
afebb4|OK  |   921KiB/s|/etc/listbot/iprep.yaml.bz2

Status Legend:
(OK):download completed.

Distributed T-Pot setup, sending T-Pot logs to yyyyyy.XXXXXXXX.net.

T-Pot type: SENSOR
Keyfile used: /data/elk/logstash/Sidney-Australia
Hive username: Sidney-Australia
Hive IP: yyyyyy.XXXXXXXX.net

Connection to Listbot looks good, now downloading latest translation maps.

05/03 20:19:52 [NOTICE] Downloading 1 item(s)
[#568597 0B/23KiB(0%) CN:1 DL:30KiB]

05/03 20:19:53 [NOTICE] Download complete: /etc/listbot/cve.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
568597|OK  |    90KiB/s|/etc/listbot/cve.yaml.bz2

Status Legend:
(OK):download completed.

05/03 20:19:53 [NOTICE] Downloading 1 item(s)
[#b3c500 0B/1.6MiB(0%) CN:1 DL:31KiB ETA:54s]
[#b3c500 400KiB/1.6MiB(23%) CN:1 DL:332KiB ETA:3s]

05/03 20:19:56 [NOTICE] Download complete: /etc/listbot/iprep.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
b3c500|OK  |   0.9MiB/s|/etc/listbot/iprep.yaml.bz2

Status Legend:
(OK):download completed.

Distributed T-Pot setup, sending T-Pot logs to yyyyyy.XXXXXXXX.net.

T-Pot type: SENSOR
Keyfile used: /data/elk/logstash/Sidney-Australia
Hive username: Sidney-Australia
Hive IP: yyyyyy.XXXXXXXX.net

Connection to Listbot looks good, now downloading latest translation maps.

05/03 20:20:58 [NOTICE] Downloading 1 item(s)
[#dfcd8d 0B/23KiB(0%) CN:1 DL:30KiB]

05/03 20:20:59 [NOTICE] Download complete: /etc/listbot/cve.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
dfcd8d|OK  |    91KiB/s|/etc/listbot/cve.yaml.bz2

Status Legend:
(OK):download completed.

05/03 20:20:59 [NOTICE] Downloading 1 item(s)
[#414006 0B/1.6MiB(0%) CN:1 DL:29KiB ETA:57s]
[#414006 352KiB/1.6MiB(20%) CN:1 DL:278KiB ETA:4s]

05/03 20:21:02 [NOTICE] Download complete: /etc/listbot/iprep.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
414006|OK  |   0.9MiB/s|/etc/listbot/iprep.yaml.bz2

Status Legend:
(OK):download completed.

Distributed T-Pot setup, sending T-Pot logs to yyyyyy.XXXXXXXX.net.

T-Pot type: SENSOR
Keyfile used: /data/elk/logstash/Sidney-Australia
Hive username: Sidney-Australia
Hive IP: yyyyyy.XXXXXXXX.net

Connection to Listbot looks good, now downloading latest translation maps.

05/03 20:22:03 [NOTICE] Downloading 1 item(s)
[#878696 0B/0B CN:1 DL:0B]

05/03 20:22:04 [NOTICE] Download complete: /etc/listbot/cve.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
878696|OK  |    90KiB/s|/etc/listbot/cve.yaml.bz2

Status Legend:
(OK):download completed.

05/03 20:22:04 [NOTICE] Downloading 1 item(s)
[#ea84a0 0B/1.6MiB(0%) CN:1 DL:30KiB ETA:56s]
[#ea84a0 384KiB/1.6MiB(22%) CN:1 DL:302KiB ETA:4s]

05/03 20:22:07 [NOTICE] Download complete: /etc/listbot/iprep.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
ea84a0|OK  |   0.9MiB/s|/etc/listbot/iprep.yaml.bz2

Status Legend:
(OK):download completed.

Distributed T-Pot setup, sending T-Pot logs to yyyyyy.XXXXXXXX.net.

T-Pot type: SENSOR
Keyfile used: /data/elk/logstash/Sidney-Australia
Hive username: Sidney-Australia
Hive IP: yyyyyy.XXXXXXXX.net

Connection to Listbot looks good, now downloading latest translation maps.

05/03 20:23:09 [NOTICE] Downloading 1 item(s)
[#74c2e9 0B/23KiB(0%) CN:1 DL:29KiB]

05/03 20:23:10 [NOTICE] Download complete: /etc/listbot/cve.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
74c2e9|OK  |    88KiB/s|/etc/listbot/cve.yaml.bz2

Status Legend:
(OK):download completed.

05/03 20:23:10 [NOTICE] Downloading 1 item(s)
[#c3d52f 0B/1.6MiB(0%) CN:1 DL:31KiB ETA:55s]
[#c3d52f 416KiB/1.6MiB(24%) CN:1 DL:336KiB ETA:3s]

05/03 20:23:12 [NOTICE] Download complete: /etc/listbot/iprep.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
c3d52f|OK  |   1.1MiB/s|/etc/listbot/iprep.yaml.bz2

Status Legend:
(OK):download completed.

Distributed T-Pot setup, sending T-Pot logs to yyyyyy.XXXXXXXX.net.

T-Pot type: SENSOR
Keyfile used: /data/elk/logstash/Sidney-Australia
Hive username: Sidney-Australia
Hive IP: yyyyyy.XXXXXXXX.net

Connection to Listbot looks good, now downloading latest translation maps.

05/03 20:24:13 [NOTICE] Downloading 1 item(s)
[#ac6f05 0B/23KiB(0%) CN:1 DL:31KiB]

05/03 20:24:14 [NOTICE] Download complete: /etc/listbot/cve.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
ac6f05|OK  |    91KiB/s|/etc/listbot/cve.yaml.bz2

Status Legend:
(OK):download completed.

05/03 20:24:14 [NOTICE] Downloading 1 item(s)
[#313301 0B/1.6MiB(0%) CN:1 DL:30KiB ETA:55s]
[#313301 400KiB/1.6MiB(23%) CN:1 DL:319KiB ETA:4s]

05/03 20:24:17 [NOTICE] Download complete: /etc/listbot/iprep.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
313301|OK  |   0.9MiB/s|/etc/listbot/iprep.yaml.bz2

Status Legend:
(OK):download completed.

Distributed T-Pot setup, sending T-Pot logs to yyyyyy.XXXXXXXX.net.

T-Pot type: SENSOR
Keyfile used: /data/elk/logstash/Sidney-Australia
Hive username: Sidney-Australia
Hive IP: yyyyyy.XXXXXXXX.net

Connection to Listbot looks good, now downloading latest translation maps.

05/03 20:25:19 [NOTICE] Downloading 1 item(s)
[#4b1148 0B/23KiB(0%) CN:1 DL:30KiB]

05/03 20:25:20 [NOTICE] Download complete: /etc/listbot/cve.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
4b1148|OK  |    91KiB/s|/etc/listbot/cve.yaml.bz2

Status Legend:
(OK):download completed.

05/03 20:25:20 [NOTICE] Downloading 1 item(s)
[#86cff6 0B/1.6MiB(0%) CN:1 DL:31KiB ETA:54s]
[#86cff6 416KiB/1.6MiB(24%) CN:1 DL:345KiB ETA:3s]

05/03 20:25:22 [NOTICE] Download complete: /etc/listbot/iprep.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
86cff6|OK  |   0.9MiB/s|/etc/listbot/iprep.yaml.bz2

Status Legend:
(OK):download completed.

Distributed T-Pot setup, sending T-Pot logs to yyyyyy.XXXXXXXX.net.

T-Pot type: SENSOR
Keyfile used: /data/elk/logstash/Sidney-Australia
Hive username: Sidney-Australia
Hive IP: yyyyyy.XXXXXXXX.net

Connection to Listbot looks good, now downloading latest translation maps.

05/03 20:26:24 [NOTICE] Downloading 1 item(s)
[#544c74 0B/23KiB(0%) CN:1 DL:31KiB]

05/03 20:26:25 [NOTICE] Download complete: /etc/listbot/cve.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
544c74|OK  |    92KiB/s|/etc/listbot/cve.yaml.bz2

Status Legend:
(OK):download completed.

05/03 20:26:25 [NOTICE] Downloading 1 item(s)
[#df3ea2 0B/1.6MiB(0%) CN:1 DL:30KiB ETA:57s]
[#df3ea2 416KiB/1.6MiB(24%) CN:1 DL:326KiB ETA:4s]

05/03 20:26:27 [NOTICE] Download complete: /etc/listbot/iprep.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
df3ea2|OK  |   1.0MiB/s|/etc/listbot/iprep.yaml.bz2

Status Legend:
(OK):download completed.

Distributed T-Pot setup, sending T-Pot logs to yyyyyy.XXXXXXXX.net.

T-Pot type: SENSOR
Keyfile used: /data/elk/logstash/Sidney-Australia
Hive username: Sidney-Australia
Hive IP: yyyyyy.XXXXXXXX.net

Connection to Listbot looks good, now downloading latest translation maps.

05/03 20:27:29 [NOTICE] Downloading 1 item(s)
[#c84331 0B/23KiB(0%) CN:1 DL:30KiB]

05/03 20:27:30 [NOTICE] Download complete: /etc/listbot/cve.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
c84331|OK  |    90KiB/s|/etc/listbot/cve.yaml.bz2

Status Legend:
(OK):download completed.

05/03 20:27:30 [NOTICE] Downloading 1 item(s)
[#6db3df 0B/1.6MiB(0%) CN:1 DL:30KiB ETA:56s]
[#6db3df 416KiB/1.6MiB(24%) CN:1 DL:335KiB ETA:3s]

05/03 20:27:33 [NOTICE] Download complete: /etc/listbot/iprep.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
6db3df|OK  |   1.0MiB/s|/etc/listbot/iprep.yaml.bz2

Status Legend:
(OK):download completed.

Distributed T-Pot setup, sending T-Pot logs to yyyyyy.XXXXXXXX.net.

T-Pot type: SENSOR
Keyfile used: /data/elk/logstash/Sidney-Australia
Hive username: Sidney-Australia
Hive IP: yyyyyy.XXXXXXXX.net

Connection to Listbot looks good, now downloading latest translation maps.

05/03 20:28:34 [NOTICE] Downloading 1 item(s)
[#2bb1cb 0B/23KiB(0%) CN:1 DL:31KiB]

05/03 20:28:35 [NOTICE] Download complete: /etc/listbot/cve.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
2bb1cb|OK  |    93KiB/s|/etc/listbot/cve.yaml.bz2

Status Legend:
(OK):download completed.

05/03 20:28:35 [NOTICE] Downloading 1 item(s)
[#b39227 0B/1.6MiB(0%) CN:1 DL:29KiB ETA:57s]
[#b39227 416KiB/1.6MiB(24%) CN:1 DL:319KiB ETA:4s]

05/03 20:28:38 [NOTICE] Download complete: /etc/listbot/iprep.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
b39227|OK  |   1.0MiB/s|/etc/listbot/iprep.yaml.bz2

Status Legend:
(OK):download completed.

Distributed T-Pot setup, sending T-Pot logs to yyyyyy.XXXXXXXX.net.

T-Pot type: SENSOR
Keyfile used: /data/elk/logstash/Sidney-Australia
Hive username: Sidney-Australia
Hive IP: yyyyyy.XXXXXXXX.net

Connection to Listbot looks good, now downloading latest translation maps.

05/03 20:29:39 [NOTICE] Downloading 1 item(s)
[#3b6dd7 0B/0B CN:1 DL:0B]

05/03 20:29:40 [NOTICE] Download complete: /etc/listbot/cve.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
3b6dd7|OK  |    91KiB/s|/etc/listbot/cve.yaml.bz2

Status Legend:
(OK):download completed.

05/03 20:29:40 [NOTICE] Downloading 1 item(s)
[#6281c6 0B/1.6MiB(0%) CN:1 DL:29KiB ETA:58s]
[#6281c6 400KiB/1.6MiB(23%) CN:1 DL:309KiB ETA:4s]

05/03 20:29:43 [NOTICE] Download complete: /etc/listbot/iprep.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
6281c6|OK  |   0.9MiB/s|/etc/listbot/iprep.yaml.bz2

Status Legend:
(OK):download completed.

Distributed T-Pot setup, sending T-Pot logs to yyyyyy.XXXXXXXX.net.

T-Pot type: SENSOR
Keyfile used: /data/elk/logstash/Sidney-Australia
Hive username: Sidney-Australia
Hive IP: yyyyyy.XXXXXXXX.net

Connection to Listbot looks good, now downloading latest translation maps.

05/03 20:30:45 [NOTICE] Downloading 1 item(s)
[#d3ba85 0B/23KiB(0%) CN:1 DL:30KiB]

05/03 20:30:46 [NOTICE] Download complete: /etc/listbot/cve.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
d3ba85|OK  |    88KiB/s|/etc/listbot/cve.yaml.bz2

Status Legend:
(OK):download completed.

05/03 20:30:46 [NOTICE] Downloading 1 item(s)
[#798a6f 0B/1.6MiB(0%) CN:1 DL:29KiB ETA:58s]
[#798a6f 400KiB/1.6MiB(23%) CN:1 DL:303KiB ETA:4s]

05/03 20:30:49 [NOTICE] Download complete: /etc/listbot/iprep.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
798a6f|OK  |   921KiB/s|/etc/listbot/iprep.yaml.bz2

Status Legend:
(OK):download completed.

Distributed T-Pot setup, sending T-Pot logs to yyyyyy.XXXXXXXX.net.

T-Pot type: SENSOR
Keyfile used: /data/elk/logstash/Sidney-Australia
Hive username: Sidney-Australia
Hive IP: yyyyyy.XXXXXXXX.net

Connection to Listbot looks good, now downloading latest translation maps.

05/03 20:31:51 [NOTICE] Downloading 1 item(s)
[#273be1 0B/23KiB(0%) CN:1 DL:30KiB]

05/03 20:31:52 [NOTICE] Download complete: /etc/listbot/cve.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
273be1|OK  |    90KiB/s|/etc/listbot/cve.yaml.bz2

Status Legend:
(OK):download completed.

05/03 20:31:52 [NOTICE] Downloading 1 item(s)
[#df7d8b 0B/1.6MiB(0%) CN:1 DL:30KiB ETA:55s]
[#df7d8b 400KiB/1.6MiB(23%) CN:1 DL:324KiB ETA:4s]

05/03 20:31:54 [NOTICE] Download complete: /etc/listbot/iprep.yaml.bz2

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
df7d8b|OK  |   0.9MiB/s|/etc/listbot/iprep.yaml.bz2

Status Legend:
(OK):download completed.

Distributed T-Pot setup, sending T-Pot logs to yyyyyy.XXXXXXXX.net.

T-Pot type: SENSOR
Keyfile used: /data/elk/logstash/Sidney-Australia
Hive username: Sidney-Australia
Hive IP: yyyyyy.XXXXXXXX.net

**Reply by rocbr** [reply]
The system was working and stopped yesterday, probably after the docker image auto update by crontab.

**Reply by t3chn0m4g3** [reply]
@rocbr Thanks. When you run `dps.sh` does it complain about health issues on elastic, logstash or kibana? Your log seems to be stuck in a loop where logstash is not even starting the process which would indicate a elasticsearch health issue.

**Reply by t3chn0m4g3** [reply]
Thanks. The logs helped a lot. Working on the fix now.

**Reply by t3chn0m4g3** [reply, marked as answer]
@enzodevs2000  @rocbr
See details in #1320. New image has been built and pushed. With `systemctl stop tpot && docker pull dtagdevsec/logstash:2204 && systemctl start tpot` T-Pot will be stopped, the new logstash image downloaded and T-Pot started again. Please let me know if this fixes all the reported issues.
Thanks again for reporting!

**Reply by rocbr** [reply]
Logstash is working like a charm !

Thanks

**Reply by t3chn0m4g3** [reply]
Awesome! Thanks for testing!

**Comment by rocbr** [comment]
dps.sh
[ ========| System |======== ]
     DATE:  Wed May  3 21:01:10 UTC 2023
   UPTIME:   21:01:10 up 17:17,  1 user,  load average: 0.30, 0.44, 0.52
    T-POT:  ACTIVE       
BLACKHOLE:  DISABLED

NAME                  STATUS                       PORTS
adbhoney              Up 7 hours (healthy)   0.0.0.0:5555->5555/tcp
ciscoasa              Up 7 hours             0.0.0.0:5000->5000/udp, 0.0.0.0:8443->8443/tcp
citrixhoneypot        Up 7 hours             0.0.0.0:443->443/tcp
conpot_guardian_ast   Up 7 hours (healthy)   0.0.0.0:10001->10001/tcp
conpot_iec104         Up 7 hours (healthy)   0.0.0.0:161->161/udp, 0.0.0.0:2404->2404/tcp
conpot_ipmi           Up 7 hours (healthy)   0.0.0.0:623->623/udp
conpot_kamstrup_382   Up 7 hours (healthy)   0.0.0.0:1025->1025/tcp, 0.0.0.0:50100->50100/tcp
cowrie                Up 7 hours             0.0.0.0:22-23->22-23/tcp
ddospot               Up 7 hours             0.0.0.0:19->19/udp, 0.0.0.0:53->53/udp, 0.0.0.0:123->123/udp, 0.0.0.0:1900->1900/udp
dicompot              Up 7 hours             0.0.0.0:11112->11112/tcp
dionaea               Up 7 hours (healthy)   0.0.0.0:20-21->20-21/tcp, 0.0.0.0:42->42/tcp, 0.0.0.0:81->81/tcp, 0.0.0.0:135->135/tcp, 0.0.0.0:445->445/tcp, 0.0.0.0:1433->1433/tcp, 0.0.0.0:1723->1723/tcp, 0.0.0.0:1883->1883/tcp, 0.0.0.0:3306->3306/tcp, 0.0.0.0:27017->27017/tcp, 0.0.0.0:69->69/udp
elasticpot            Up 7 hours             0.0.0.0:9200->9200/tcp
fatt                  Up 7 hours             
heralding             Up 7 hours             0.0.0.0:110->110/tcp, 0.0.0.0:143->143/tcp, 0.0.0.0:465->465/tcp, 0.0.0.0:993->993/tcp, 0.0.0.0:995->995/tcp, 0.0.0.0:1080->1080/tcp, 0.0.0.0:5432->5432/tcp, 0.0.0.0:5900->5900/tcp
honeytrap             Up 7 hours             
ipphoney              Up 7 hours             0.0.0.0:631->631/tcp
mailoney              Up 7 hours             0.0.0.0:25->25/tcp
medpot                Up 7 hours             0.0.0.0:2575->2575/tcp
p0f                   Up 7 hours             
redishoneypot         Up 7 hours             0.0.0.0:6379->6379/tcp
sentrypeer            Up 7 hours             0.0.0.0:5060->5060/udp
snare                 Up 7 hours             0.0.0.0:80->80/tcp
suricata              Up 7 hours             
tanner                Up 7 hours             
tanner_api            Up 7 hours             
tanner_phpox          Up 7 hours             
tanner_redis          Up 7 hours             
logstash              **DOWN**

**Comment by rocbr** [comment]
glances:

...
r-c9458819f07b            0b     0b    fatt                  running   6 hours   0.0  12.6M/7.57G       0B 0B           _ _       /bin/sh -c python3 fatt.py -i $(/sbin/ip addre
br-ce22001439d8            0b     0b    heralding             running   6 hours   0.0  29.4M/7.57G       0B 0B        840b 816b    /bin/sh -c exec heralding -c /etc/heralding/he
br-dfbbf3e81132           7Kb   12Kb    honeytrap             running   6 hours   0.1  36.0M/7.57G       0B 0B           _ _       /opt/honeytrap/sbin/honeytrap -D -C /opt/honey
br-f1ea71a50242            0b     0b    ipphoney              running   6 hours   0.0  16.2M/7.57G       0B 0B          0b 0b      /usr/bin/python3 ipphoney.py
br-f667bd6136ca            0b     0b    logstash           **restarting**         _     _      _/_            _ _            _ _       /bin/sh -c entrypoint.sh
eth0                   1.03Mb 1.26Mb    mailoney              running   6 hours   0.0  6.18M/7.57G       0B 0B          0b 0b      /usr/bin/python mailoney.py -i 0.

...

**Comment by rocbr** [comment]
At Hive server everything seems ok:

dps.sh
[ ========| System |======== ]
     DATE:  Wed May  3 18:05:50 -03 2023
   UPTIME:   18:05:50 up  3:03,  1 user,  load average: 0.14, 0.24, 0.23
    T-POT:  ACTIVE       
BLACKHOLE:  DISABLED

NAME                  STATUS                       PORTS
elasticsearch   Up 3 hours (healthy)   127.0.0.1:64298->9200/tcp
kibana          Up 3 hours (healthy)   127.0.0.1:64296->5601/tcp
logstash        Up 3 hours (healthy)   127.0.0.1:64305->64305/tcp
map_data        Up 46 seconds          
map_redis       Up 47 seconds          
map_web         Up 47 seconds          127.0.0.1:64299->64299/tcp
nginx           Up 3 hours             
spiderfoot      Up 3 hours (healthy)   127.0.0.1:64303->8080/tcp

**Comment by rocbr** [comment]
But as @enzodevs2000 said, when running the command at the HIVE
**ps -aux | grep sshd**

I should see the ssh connections from the SENSORs , but none appear, which means that logstash docker image is not creating the ssh tunnel through autossh, which reinforces the thesis that this docker image has a problem.
